SPECS/distroless-packages/distroless-packages.spec

```diff
@@ -14,7 +14,7 @@ Metapackage holding sets of core packages for different applications.
 %package minimal
 Summary:        The smallest useful package list.
 Requires:       filesystem
-Requires:       mariner-release
+Requires:       azurelinux-release
```

Comment:
Ideally, could you bump the `Release` tag for all modified specs, since these change the "behaviour" of the spec?

Reply:
Then we would also want a changelog entry for all of them too

Reply:
Yes, a bump will require a changelog entry.

Reply:
Updated